### PR TITLE
Add problem matchers for RuboCop and minitest

### PR DESCRIPTION
**Pull request by tido64**

### CODE REVIEW

File: .github/minitest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "pattern": [
+        {
+          "regexp": "^.*?\\[(.*?):(\\d+)\\]:$",
+          "file": 1,
+          "line": 2
+        },
+        {
+          "regexp": "^\\s*((?:Expected|Actual).*)$",
+          "message": 1,
+          "loop": true
```
Comment (tido64) [inline .github/minitest.json:14]: Due to a limitation in the format of problem matchers, it's currently not possible to extract multiline messages. For input such as:
```
TestTestApp#test_resources_pod_returns_spec_path [test/test_test_app.rb:31]:
Expected: ".."
  Actual: "."
```
"Expected: …" and "Actual: …" are two separate errors with this matcher. We are currently not able to match output such as:
```
TestTestApp#test_resources_pod_writes_podspec [test/test_test_app.rb:50]:
--- expected
+++ actual
@@ -1 +1 @@
-["dist/assetd", "dist/main.jsbundle"]
+["dist/assets", "dist/main.jsbundle"]
```
This issue has been reported in https://github.com/actions/toolkit/issues/193 and https://github.com/actions/toolkit/issues/319.

**Comment by tido64** [comment]
Example output:
Screenshot 2020-04-09 at 20 03 14
Screenshot 2020-04-09 at 20 02 23
Screenshot 2020-04-09 at 20 01 55

**Comment by ArazAbishov** [comment]
Should we add a rule to require RuboCop / ESLint checks to pass before merging PRs?

**Comment by tido64** [comment]
> Should we add a rule to require RuboCop / ESLint checks to pass before merging PRs?

Yeah, I can make them required.

**Pull request merged**